bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/GemsStrategy.scala

```diff
@@ -172,27 +175,27 @@ trait GemsStrategy extends AgsStrategy {
       val odgwMagLimits = (lim(GsaoiOdgw.odgw1) /: GsaoiOdgw.values().drop(1)) { (ml, odgw) =>
         (ml |@| lim(odgw))(_ union _).flatten
       }
-      val canMagLimits = (lim(Canopus.Wfs.cwfs1) /: Canopus.Wfs.values().drop(1)) { (ml, can) =>
+      val canMagLimits = (lim(CanopusWfs.cwfs1) /: CanopusWfs.values().drop(1)) { (ml, can) =>
         (ml |@| lim(can))(_ union _).flatten
       }
 
-      val canopusConstraint = canMagLimits.map(c => CatalogQuery(CanopusTipTiltId, base.toNewModel, RadiusConstraint.between(Angle.zero, Canopus.Wfs.Group.instance.getRadiusLimits.toNewModel), List(ctx.getConditions.adjust(c)), UCAC4))
-      val odgwConstraint    = odgwMagLimits.map(c => CatalogQuery(OdgwFlexureId,   base.toNewModel, RadiusConstraint.between(Angle.zero, GsaoiOdgw.Group.instance.getRadiusLimits.toNewModel), List(ctx.getConditions.adjust(c)), UCAC4))
+      val canopusConstraint = canMagLimits.map(c => CatalogQuery(CanopusTipTiltId, base.toNewModel, RadiusConstraint.between(Angle.zero, CanopusWfs.Group.instance.getRadiusLimits.toNewModel), List(ctx.getConditions.adjust(c)), PPMXL))
+      val odgwConstraint    = odgwMagLimits.map(c => CatalogQuery(OdgwFlexureId,   base.toNewModel, RadiusConstraint.between(Angle.zero, GsaoiOdgw.Group.instance.getRadiusLimits.toNewModel), List(ctx.getConditions.adjust(c)), PPMXL))
```

Comment:
More places where a specific catalog is set.

Reply:
If I recall this is from the last large catalog refactoring but can't remember the reasons. Do you think there shouldn't be a specific catalog prescribed?

Reply:
I think there are a few problems:

* If you want to change the catalog there are many places where you have to set it (6 by my count), and it seems like it would be easy to overlook one of them.
* The search for a PWFS1 star is fixed to PPMXL regardless of which catalog you use for GeMS proper, but all the selected stars should come from the same catalog.
* You can pick a catalog in the GeMS manual search tool, but I'm not 100% sure it is used because of all the places that PPMXL is explicitly set.